.github/workflows/release.yaml

```diff
@@ -21,6 +21,10 @@ jobs:
           git fetch --tags -f
       - name: Setup Go
         uses: actions/setup-go@v2
+      - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Not certain if it matters, but looking at the file and I think this block should be before the next `name:` The 
```
with:
  go-version:...
```
Seems like it should go with the Setup Go entry

Reply:
Oh wow good catch. That would not have worked.